openstates/id/bills.py

```diff
@@ -117,16 +117,16 @@ class IDBillScraper(BillScraper):
     other = False
     last_date = None
 
-
     def scrape_subjects(self, session):
+        """to scrape subjects """
```

Comment:
Not sure this docstring is useful, since it's basically the same as the method name 😄